packages/react-examples/src/react/Keytip/Keytips.Basic.Example.tsx

```diff
@@ -19,36 +20,42 @@ const sampleOptions = [
   { key: 'C', text: 'Option 3' },
 ];
 
-export const KeytipsBasicExample: React.FunctionComponent = () => (
-  <div>
-    <p>
-      For Pivots, keytips will first show for each of the pivots. After selecting a pivot, the Keytips for its content
-      are shown.
-    </p>
-    <Pivot>
-      <PivotItem headerText="Pivot 1" keytipProps={keytipMap.Pivot1Keytip} style={pivotItemStyle}>
-        <Stack tokens={stackTokens}>
-          <SpinButton label="Spin Button" keytipProps={keytipMap.SpinButtonKeytip} styles={spinButtonStyles} />
-          <Toggle onText="Yes" offText="No" keytipProps={keytipMap.ToggleKeytip} />
-          <span>
-            Go to{' '}
-            <Link keytipProps={keytipMap.LinkKeytip} href="http://www.bing.com" target="_blank">
-              Bing
-            </Link>
-          </span>
-        </Stack>
-      </PivotItem>
+export const KeytipsBasicExample: React.FunctionComponent = () => {
```

Comment:
@xugao Can you confirm that these example changes look correct? (they were copied from react-next)

Reply:
yes this is correct